package.json

```diff
@@ -35,8 +35,10 @@
     "npm": ">=5.3.0"
   },
   "devDependencies": {
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.3.1",
```

Comment:
`@babel/polyfill` and `@babel/preset-env` is'nt be necessary. Should be removed.

Reply:
yikes, my bad